package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmt-escrow",
-  "version": "0.10.2",
+  "version": "0.10.3",
```

Comment:
This is an API change. The version should be 0.11.0

Reply:
According to [semver](https://semver.org/) rules, we should change the PATCH number:

> * MAJOR version when you make incompatible API changes,
> * MINOR version when you add functionality in a backwards compatible manner, and
> * PATCH version when you make backwards compatible bug fixes.

Reply:
Oh sure! Doing it!